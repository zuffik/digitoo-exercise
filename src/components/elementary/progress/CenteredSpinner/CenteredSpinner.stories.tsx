import {CenteredSpinner} from './CenteredSpinner';

export default {
    title: 'Elementary/Progress/CenteredSpinner',
};

export const centeredSpinner = CenteredSpinner;
