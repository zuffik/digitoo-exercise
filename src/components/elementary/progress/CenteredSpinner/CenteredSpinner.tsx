import * as React from 'react';
import {Spinner} from '../Spinner/Spinner';

interface Props {}

export const CenteredSpinner: React.FC<Props> = (props: Props) => {
    return (
        <div className="d-flex justify-content-center">
            <Spinner />
        </div>
    );
};
