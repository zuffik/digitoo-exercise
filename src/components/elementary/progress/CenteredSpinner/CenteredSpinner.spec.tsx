import * as React from 'react';
import {render} from '@testing-library/react'
import {CenteredSpinner} from "./CenteredSpinner";

describe('<CenteredSpinner/>', () => {
    it('should render', () => {
        const {asFragment} = render(<CenteredSpinner/>);
        expect(asFragment()).toMatchSnapshot();
    });
});
