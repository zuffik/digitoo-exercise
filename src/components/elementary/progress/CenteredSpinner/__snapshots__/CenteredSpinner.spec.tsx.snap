// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CenteredSpinner/> should render 1`] = `
<DocumentFragment>
  <div
    class="d-flex justify-content-center"
  >
    <div
      class="spinner-border"
      style="width: 48px; height: 48px;"
    />
  </div>
</DocumentFragment>
`;
